### PR TITLE
Report actual amounts added in LiquidityAdded event

### PR DESCRIPTION
**Pull request by benluelo**

### CODE REVIEW

**Review by PoisonPhang — APPROVED** [review]

File: code/parachain/frame/pablo/src/dual_asset_constant_product.rs [hunk 1]
```diff
@@ -139,7 +141,13 @@ impl<T: Config> DualAssetConstantProduct<T> {
 				keep_alive,
 			)?;
 
-			single_deposit.value
+			(
+				single_deposit.value,
+				assets_with_balances
```
Comment (vimukthi-git) [inline code/parachain/frame/pablo/src/dual_asset_constant_product.rs:146]: NIT: Additional iterations to be removed

**Review by andresmechali — APPROVED** [review]

**Review by KaiserKarel — APPROVED** [review]

File: code/parachain/frame/pablo/src/dual_asset_constant_product.rs [hunk 2]
```diff
@@ -162,7 +170,13 @@ impl<T: Config> DualAssetConstantProduct<T> {
 					)?;
 				}
 
-				lp_to_mint
+				(
+					lp_to_mint,
+					assets_with_balances
```
Comment (vimukthi-git) [inline code/parachain/frame/pablo/src/dual_asset_constant_product.rs:175]: NIT: Additional iterations to be removed

File: code/parachain/frame/pablo/src/test/dual_asset_constant_product_tests.rs [hunk 3]
```diff
@@ -1136,3 +1136,84 @@ mod twap {
...
+		);
+
+		dbg!();
+
+		// Mint the tokens
+		assert_ok!(Tokens::mint_into(FIRST_ASSET::ID, &BOB, first_asset_amount));
+		assert_ok!(Tokens::mint_into(SECOND_ASSET::ID, &BOB, second_asset_amount));
+
+		Test::assert_extrinsic_event(
+			Pablo::add_liquidity(Origin::signed(BOB), pool_id, assets_with_amounts, 0, false),
```
Comment (vimukthi-git) [inline code/parachain/frame/pablo/src/test/dual_asset_constant_product_tests.rs:1207]: :+1:

**Review by vimukthi-git — APPROVED** [review]
LGTM :rocket:

File: code/parachain/frame/pablo/src/dual_asset_constant_product.rs (hunk 1 above)
Comment (benluelo) [reply code/parachain/frame/pablo/src/dual_asset_constant_product.rs:146]: what do you mean by iterations? `assets_with_balances` is a `Vec<AssetAmount>` and we need a `BTreeMap<AssetId, Balance>`

**Review by KaiserKarel — APPROVED** [review]

**Pull request merged**